client/lib/pages/main_pages/check_up_intro_page.dart

```diff
@@ -0,0 +1,166 @@
+import 'package:who_app/components/page_scaffold/page_header.dart';
+import 'package:who_app/components/page_scaffold/page_scaffold.dart';
+import 'package:who_app/components/themed_text.dart';
+import 'package:who_app/constants.dart';
+
+class CheckUpIntroPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    final inHomePage = !ModalRoute.of(context).canPop;
+    return PageScaffold(
+      showBackButton: !inHomePage,
+      color: CupertinoColors.white,
+      // TODO: localize
+      title: 'Check-Up',
+      header: inHomePage
+          ? SliverSafeArea(
+              top: true,
+              bottom: false,
+              sliver: SliverToBoxAdapter(
+                child: Padding(
+                  padding: EdgeInsets.fromLTRB(16, 36, 16, 20),
+                  // TODO: localize
+                  child: PageHeader.buildTitle(
+                    'Check-Up',
+                    textStyle:
+                        ThemedText.styleForVariant(TypographyVariant.title)
+                            .merge(
+                      TextStyle(color: Constants.primaryDarkColor),
+                    ),
+                  ),
+                ),
+              ),
+            )
+          : null,
+      body: <Widget>[
+        if (!inHomePage) SliverToBoxAdapter(child: SizedBox(height: 24)),
+        _buildBody(context)
+      ],
+    );
+  }
+
+  Widget _buildBody(BuildContext context) => SliverList(
+          delegate: SliverChildListDelegate([
+        ...optionWidgets(),
+        submitWidget(context),
+      ]));
+
+  List<Widget> optionWidgets() {
+    // TODO: localize everything
+    return <Widget>[
+      Padding(
+          padding: const EdgeInsets.only(
+            left: 24,
+            right: 60,
+            bottom: 20,
+            top: 0,
+          ),
+          child: ThemedText(
+            'You should know...',
+            variant: TypographyVariant.body,
+            style: TextStyle(fontWeight: FontWeight.bold),
+          )),
+      _ListItem(
+          title:
+              'Your answers will not be shared with WHO or others without your permission.',
+          iconName: 'safe'),
+      _ListItem(
+        title:
+            'By using this tool, you agree to its terms and that WHO will not be liable for any harm relating to your use.',
+        iconName: 'check',
+      ),
+      _ListItem(
+        title:
+            'Information provided by this tool does not constitute medical advise and should not be used to diagnose or treat medical conditions.',
+        iconName: 'medical',
+        extra: CupertinoButton(
+          padding: EdgeInsets.zero,
+          child: ThemedText(
+            'See terms ›',
+            variant: TypographyVariant.body,
+            style: TextStyle(
+              color: Constants.whoBackgroundBlueColor,
+            ),
+          ),
+          onPressed: () {
+            return launch('https://whoapp.org/terms');
```

Comment:
WRONG LINK!!! ;-)

Please use this logic instead:
https://github.com/WorldHealthOrganization/app/blob/8bb78d7f9c30d75805e4b1bdcf3cac3607bd6a33/client/lib/pages/about_page.dart#L33-L37